### PR TITLE
Support tuples(only primitives) from Swift -> Rust

### PR DESCRIPTION
**Pull request by NiwakaDev**
This commit adds support tuples from Swift -> Rust, which consist of only primitives.

related to #204.

Here's an example of using this:

```rust
#[swift_bridge::bridge]
mod ffi {
    extern "Swift" {
        fn swift_reflect_tuple_primitives(arg: (i32, u32)) -> (i32, u32);
    }
}
```

```rust
//Rust

//... 
let tuple = swift_reflect_tuple_primitives((-123, 123));
//...
```

### CODE REVIEW

File: SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/TupleTests.swift
Comment (chinedufn) [inline SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/TupleTests.swift:33]: `testRustCallsSwiftTuple`

**Review by chinedufn — APPROVED** [review]
Looks great. Just need to rename one of the tests.

Feel free to merge this yourself once you fix the test name. You can squash down to a single commit and use your PR body as the commit message. You can say `This commit` instead of `This PR`.

Thanks!

**Comment by chinedufn** [comment]
Looks like we merged 2 commits instead of 1 and the commit doesn't have a link to the PR.

I'll rebase

**Comment by NiwakaDev** [comment]
>Looks like we merged 2 commits instead of 1 and the commit doesn't have a link to the PR.

Sorry... But This PR seems to have one commit. Why did 2 commits ged merged?

**Comment by chinedufn** [comment]
No problem at all! Not sure either.

**Comment by NiwakaDev** [comment]
>OR use GitHub's Squash and Merge button

A new commit containing only a merge has been added because I used only Merge button.

Here're my commits, which are related to this PR.
024deccfe3b0760e4b7de8e303164f4a3a7e7f11       (merge commit)
be4c8b64d892c342ad0eb7be06c25c503ebaab2b  (rebased commit)

So, next time I'll use Squash and Merge button.

**Pull request merged**
